src/components/MovieCard/MovieCard.tsx

```diff
@@ -6,39 +6,61 @@ type Props = {
   movie: Movie,
 };
 
-export const MovieCard: React.FC<Props> = ({ movie }) => (
-  <div className="card" data-cy="movieCard">
-    <div className="card-image">
-      <figure className="image is-4by3">
-        <img
-          data-cy="moviePoster"
-          src={movie.imgUrl}
-          alt="Film logo"
-        />
-      </figure>
-    </div>
-    <div className="card-content">
-      <div className="media">
-        <div className="media-left">
-          <figure className="image is-48x48">
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    imgUrl,
+    title,
+    description,
+    imdbUrl,
+  } = movie;
+
+  const isTitle = (imgUrl !== 'N/A');
+  // why we should use image from another server, but not from this program?
+  // const posterDef = 'images/noPreview.png';
+  const posterDef = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+  return (
+    <div className="card" data-cy="movieCard">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          {isTitle ? (
             <img
-              src="images/imdb-logo.jpeg"
-              alt="imdb"
+              data-cy="moviePoster"
+              src={imgUrl}
+              alt="Film logo"
             />
-          </figure>
-        </div>
-        <div className="media-content">
-          <p className="title is-8" data-cy="movieTitle">{movie.title}</p>
-        </div>
+          ) : (
+            <img
+              data-cy="moviePoster"
+              src={posterDef}
```

Comment:
You can use ternary operator just for `src` attribute instead of rendering two different `img`:
```
<img
  data-cy="moviePoster"
  src={isTitle ? imgUrl : posterDef}
  alt="Film logo"
/>
```